build-system/release-workflows/upload-release.js

```diff
@@ -100,6 +101,54 @@ function logProgress_(totalFiles, uploadedFiles) {
   }
 }
 
+/**
+ * Compresses a single file with Brotli and writes it to ${path}.br.
+ * @param {string} path
+ * @param {number} sizeHint
+ * @return {Promise<void>}
+ */
+async function brotliCompressFile_(path, sizeHint) {
+  const fileBuffer = fs.readFileSync(path);
+  const compressedBuffer = zlib.brotliCompressSync(fileBuffer, {
+    params: {
+      [zlib.constants.BROTLI_PARAM_QUALITY]: zlib.constants.BROTLI_MAX_QUALITY,
+      [zlib.constants.BROTLI_PARAM_SIZE_HINT]: sizeHint,
+    },
```

Comment:
List of possible params: https://nodejs.org/docs/latest-v16.x/api/zlib.html#compressor-options